api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java

```diff
@@ -0,0 +1,1843 @@
+                    }
+                    displayMotTestItem.setFuelType(resultSet.getString(8));
+                    displayMotTestItem.setPrimaryColour(resultSet.getString(9));
+                    // secondary colour not used
+                    // started date not used
+                    if (resultSet.getDate(12) != null) {
+                        displayMotTestItem.setCompletedDate(sdfDateTime.format(resultSet.getTimestamp(12)));
+                    }
+                    displayMotTestItem.setTestResult(resultSet.getString(13));
+                    if (resultSet.getDate(14) != null) {
+                        displayMotTestItem.setExpiryDate(sdfDate.format(resultSet.getTimestamp(14)));
+                    }
+                    displayMotTestItem.setOdometerValue(String.valueOf(resultSet.getInt(15)));
+                    displayMotTestItem.setOdometerUnit(resultSet.getString(16));
+                    // odometer result type not used
+                    displayMotTestItem.setMotTestNumber(String.valueOf(resultSet.getBigDecimal(18)));
+                }
+
+                if (currMotTestRfrMapId != 0) {
+                    RfrAndAdvisoryItem rfrAndAdvisoryItem = new RfrAndAdvisoryItem();
+                    rfrAndAdvisoryItem.setType(resultSet.getString(19));
+                    rfrAndAdvisoryItem.setText(resultSet.getString(20));
+                    rfrAndAdvisoryItems.add(rfrAndAdvisoryItem);
+                }
+            }
+
+            return displayMotTestItems;
+        } catch (Exception e) {
+            throw new InternalException(e);
+        }
+    }
+
+    private MotTest mapResultSetToMotTestCurrent(ResultSet resultSet) {
+
+        try {
+            MotTest motTestCurrent = new MotTest();
+
+            motTestCurrent.setId(resultSet.getLong(1));
+            motTestCurrent.setPerson(getPersonById(resultSet.getInt(2)));
+            motTestCurrent.setVehicleId(resultSet.getInt(3));
+            motTestCurrent.setVehicleVersion(resultSet.getInt(4));
+            motTestCurrent.setDocument(getJasperDocumentById(resultSet.getLong(5)));
+            motTestCurrent.setOrganisation(getOrganisationById(resultSet.getInt(6)));
+            motTestCurrent.setSite(getSiteById(resultSet.getInt(7)));
+            motTestCurrent.setHasRegistration(resultSet.getBoolean(8));
+            motTestCurrent.setMotTestType(getMotTestTypeById(resultSet.getInt(9)));
+            motTestCurrent.setStartedDate(resultSet.getDate(10));
+            motTestCurrent.setCompletedDate(resultSet.getDate(11));
+            motTestCurrent.setSubmittedDate(resultSet.getDate(12));
+            motTestCurrent.setMotTestStatus(getMotTestStatusById(resultSet.getInt(13)));
+            motTestCurrent.setIssuedDate(resultSet.getDate(14));
+            motTestCurrent.setExpiryDate(resultSet.getDate(15));
+            // TODO causing recursive gets because of stupid dual linking
```

Comment:
Yeah, I agree. Not a good idea to have it on a public repo either. We did talked about these comments. This wasn't in the scope of the code style changes.